.github/workflows/build-image.yml

```diff
@@ -25,8 +29,8 @@ jobs:
       - name: Get the latest tag
         id: get_tag
         run: |
-          if [[ "${{ github.event_name }}" == "push" && "${{ github.ref_type }}" == "tag" ]]; then
-            echo "tag=${{ github.ref_name }}" >> $GITHUB_OUTPUT
+          if [[ "${{ inputs.tag }}" != "" ]]; then
```

Comment:
Shouldn't we add a check for the event_name to be `workflow_call`?

Reply:
I think it's not necessary, the **inputs**.tag is only coming from workflow call.